package.json

```diff
@@ -46,14 +53,19 @@
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install",
+        "postinstall": "node ./node_modules/vscode/bin/install && curl http://try.umple.org/scripts/umple.jar --output umple.jar",
```

Comment:
download the latest umple when building/ testing. Might not work on windows shell.